packages/babel-traverse/src/index.js

```diff
@@ -90,7 +89,7 @@ traverse.hasType = function (
   blacklistTypes: Array<string>,
```

Comment:
Could you mark this type as nullable? You correctly used `blacklistTypes?.includes` because we don't always pass this parameter, but it should be reflected in the signature:

```suggestion
  blacklistTypes?: Array<string>,
```